src/nwm_routing/src/nwm_routing/preprocess.py

```diff
@@ -332,8 +331,7 @@ def nwm_forcing_preprocess(
             start_time = time.time()
         if verbose:
             print("creating usgs time_slice data array ...")
-
-        usgs_df = nnu.build_data_assimilation_usgs_df(da_run, lastobs_index, run["t0"])
+        usgs_df = nnu.build_data_assimilation_usgs_df(da_run, run, lastobs_index)
```

Comment:
lastobs_index == run['t0']?

Reply:
TypeError: Index(...) must be called with a collection of some kind, None was passed 

run['t0'] isn't sending anything.

Reply:
The difference here being that run['t0'] needs to be run as other parts of it are used downstream.